app/models/krikri/original_record.rb

```diff
@@ -42,7 +42,8 @@ def load(identifier)
           record = new(identifier)
         end
 
-        raise "No #{self} found with id: #{identifier}" unless record.exists?
+        raise NameError, "No #{self} found with id: #{identifier}" unless 
```

Comment:
[`Krikri::OriginalRecord#exists?`](https://github.com/dpla/KriKri/blob/feature/rspec-3.4/lib/krikri/ldp/resource.rb#L91-L103) is catching `Faraday::ResourceNotFound`. Is something like that preferable?